framework/src/controls/Control.C

```diff
@@ -23,9 +23,11 @@ validParams<Control>()
   params += validParams<TransientInterface>();
   params += validParams<SetupInterface>();
   params += validParams<FunctionInterface>();
+
+  params.set<ExecFlagEnum>("execute_on", true) =
+      std::vector<ExecFlagType>({EXEC_INITIAL, EXEC_TIMESTEP_END});
```

Comment:
It is just a MultiMooseEnum, people have been using MultiMooseEnum for execute flags for some time.

You can still set them as: `params.set<ExecFlagEnum>("execute_on") = "foo bar";` if you prefer, I was just trying to avoid using strings.